app/models/host/managed.rb

```diff
@@ -784,8 +784,12 @@ def power
     end
   end
 
+  def supports_power?
```

Comment:
Nice, I was wondering if you could move the power aspects of the host to a concern? `app/models/concerns/host/power_interface.rb` or something like that, then just include the concern here.

The methods from the host should be available there, it's just that this class is becoming very loaded